### DISCUSSION TITLE
How to remove the hash version number when linking to a compressed file download?

### DISCUSSION THREAD

**Discussion by KiraraWusachi** [facebook/docusaurus#9757] [Q&A] [answered]
`[Test Report](../files/Report.zip)`
Downloaded files always whith hash. How do I remove the hash.
I hope the downloaded file name is Report.zip instead of Report-HashCode.zip

**Comment by slorber** [comment]
Then you should not use a relative path but use the static folder instead: 

`static/files/Report.zip`

`[Report](/files/Report.zip)`

**Reply by KiraraWusachi** [reply]
I tried but it doesn't work

**Reply by slorber** [reply, marked as answer]
please create a repro showing it doesn't work

Also try this: `[Report](pathname:///files/Report.zip)`

**Reply by OzakIOne** [reply]
Can confirm this one works [repro](https://stackblitz.com/edit/github-w1smoq?file=docs%2Fintro.md)
